.surge.sh

```diff
@@ -0,0 +1,14 @@
+#! /bin/bash
+
+set -x
+
+if [ "$TRAVIS_EVENT_TYPE" = "push" -a "$TRAVIS_BRANCH" = "master" ]; then
```

Comment:
problem is that travis script do not have shell to execute, so it failed ( already tried ).